### PR TITLE
Recreate registration survey during migration

### PR DESCRIPTION
**Pull request by ehmadzubair**

### CODE REVIEW

**Comment by ehmadzubair** [comment]
@geoo89 I'm seeing the following missing translations in the post migration report for the registration survey:

```
===> REGISTRATION SURVEY TRANSLATION NOT FOUND
Translation not found for label: dob in locale: Ndebele
Translation not found for label: gender in locale: Ndebele
Translation not found for label: location in locale: Ndebele
Translation not found for label: mobile_number in locale: Ndebele
Translation not found for label: education_level in locale: Ndebele
Translation not found for label: email_address in locale: Ndebele
Translation not found for label: dob in locale: Chichewa
Translation not found for label: gender in locale: Chichewa
Translation not found for label: location in locale: Chichewa
Translation not found for label: mobile_number in locale: Chichewa
Translation not found for label: education_level in locale: Chichewa
Translation not found for label: email_address in locale: Chichewa
Translation not found for label: dob in locale: Shona
Translation not found for label: gender in locale: Shona
Translation not found for label: location in locale: Shona
Translation not found for label: mobile_number in locale: Shona
Translation not found for label: education_level in locale: Shona
Translation not found for label: email_address in locale: Shona
Translation not found for label: dob in locale: ta
Translation not found for label: gender in locale: ta
Translation not found for label: location in locale: ta
Translation not found for label: mobile_number in locale: ta
Translation not found for label: education_level in locale: ta
Translation not found for label: email_address in locale: ta

```

File: iogt_content_migration/management/commands/load_v1_db.py [hunk 1]
```diff
@@ -1673,10 +1680,131 @@ def move_page(self, page_to_move, position):
...
+    def migrate_post_registration_survey(self):
+
+        sql = 'select * from profiles_userprofilessettings pups ' \
+              'inner join wagtailcore_site ws on pups.site_id = ws.id ' \
+              'where is_default_site = true'
+        cur = self.db_query(sql)
+        row = cur.fetchone()
+
+        survey = Survey(
+            title='Registration Survey', live=True, allow_multiple_submissions=False, submit_button_text='Register')
```
Comment (sheralim012) [inline iogt_content_migration/management/commands/load_v1_db.py:1702]: add allow_anonymouse_submission=False

**Review by sheralim012 — CHANGES_REQUESTED** [review]

**Comment by geoo89** [comment]
This appears to crash for global:

```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/data/idems/iogt/iogt_venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/data/idems/iogt/iogt_venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/data/idems/iogt/iogt_venv/lib/python3.6/site-packages/django/core/management/base.py", line 330, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/data/idems/iogt/iogt_venv/lib/python3.6/site-packages/django/core/management/base.py", line 371, in execute
    output = self.handle(*args, **options)
  File "/data/idems/iogt/iogt/iogt_content_migration/management/commands/load_v1_db.py", line 100, in handle
    self.migrate(root)
  File "/data/idems/iogt/iogt/iogt_content_migration/management/commands/load_v1_db.py", line 190, in migrate
    self.migrate_post_registration_survey()
  File "/data/idems/iogt/iogt/iogt_content_migration/management/commands/load_v1_db.py", line 1781, in migrate_post_registration_survey
    field = SurveyFormField.objects.get(page=translated_survey, admin_label=admin_label)
  File "/data/idems/iogt/iogt_venv/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/data/idems/iogt/iogt_venv/lib/python3.6/site-packages/django/db/models/query.py", line 431, in get
    self.model._meta.object_name
questionnaires.models.DoesNotExist: SurveyFormField matching query does not exist.
```

File: iogt_content_migration/management/commands/load_v1_users.py [hunk 2]
```diff
@@ -499,6 +504,38 @@ def migrate_page_view_restrictions(self):
...
+        cur = self.db_query(sql)
+
+        site_settings = SiteSettings.get_for_default_site()
+        registration_survey = site_settings.registration_survey
+
+        if registration_survey:
+            for row in self.with_progress(sql, cur, 'User Registration Survey migration in progress'):
+
+                form_data = {
+                    'select_date_of_birth': row['date_of_birth'],
```
Comment (geoo89) [inline iogt_content_migration/management/commands/load_v1_users.py:521]: Shouldn't these match the admin labels (i.e. the same keys as in row)?

**Pull request merged**